R/air.R

```diff
@@ -0,0 +1,212 @@
+#' Configure a project to use Air
+#'
+#' @description
+#' [Air](https://github.com/posit-dev/air) is an extremely fast R code
+#' formatter. This function sets up a project to use Air. Specifically, it:
+#'
+#' - Creates an empty `air.toml` configuration file. If either an `air.toml` or
+#'   `.air.toml` file already existed, nothing is changed. If the project is an
+#'   R package, `.Rbuildignore` is updated to ignore this file.
+#'
+#' - Adds recommended settings to `.vscode/settings.json` and
+#'   `.vscode/extensions.json`. This includes:
+#'
+#'   - Setting `editor.formatOnSave = true` for R files to enable formatting on
+#'     every save.
+#'
+#'   - Setting `editor.defaultFormatter` to Air for R files to ensure that Air
+#'     is always selected as the formatter for this project.
```

Comment:
This function only sets these values if they aren't already set. Meaning that if your settings.json is

```r
{
    "[r]": {
        "editor.formatOnSave": false
    }
}
```

it will be transformed to 

```r
{
    "[r]": {
        "editor.formatOnSave": false,
        "editor.defaultFormatter": "Posit.air-vscode"
    }
}
```